src/main/java/frc/robot/commands/SpeedUpShooter.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkBase;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SparkGenericSubsystem;
+import com.spikes2212.control.FeedForwardController;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.util.UnifiedControlMode;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.Command;
+
+import java.util.function.Supplier;
+
+public class SpeedUpShooter extends Command {
```

Comment:
im assuming that this class makes the shooter move, in that case it should extend parallelcommandgroup as we want to control both flywheels at different speeds using movesmartmotorcontrollergenericsubsystem